src/views/product-tutorials-view/helpers/heading-helpers.ts

```diff
@@ -98,16 +100,11 @@ function getSitemapHeading(): TableOfContentsHeading {
 /**
  * Adds "headingSlug" properties to all blocks with defined "heading"
  */
-function addHeadingSlugsToBlocks(rawPageData: {
-	blocks: LearnClientProductPageBlock[]
-	showProductSitemap?: boolean
-}): {
-	blocks: ProductViewBlock[]
-	showProductSitemap?: boolean
-} {
-	const { blocks } = rawPageData
+function addHeadingSlugsToBlocks(
```

Comment:
This function used to accept and return full `pageData` even though it only did work on `blocks`. Have simplified it accordingly.